### ISSUE TITLE
[Feature Request] Set windows color

### ISSUE THREAD

**Issue by tjhrulz** [digimezzo/dopamine-windows#486]
Since you guys can follow windows color or album color it would be nice to have an option if following album color to set windows color. I had a project that did that in the past and the results I must say are pretty cool.

**Comment by Riajyuu** [comment]
![image](https://user-images.githubusercontent.com/16912240/29645310-46fefe90-88af-11e7-9d54-1bb06d1186a4.png)

Am I missing anything?

**Comment by EMLVIRUS** [comment]
![image](https://user-images.githubusercontent.com/16912240/29645310-46fefe90-88af-11e7-9d54-1bb06d1186a4.png)

Am I missing anything?

**Comment by tjhrulz** [comment]
You program's title bar and theming can follow it. I would like you to set the global Windows color (The one you use if you were to check "Follow the Windows color") not the color of the window. Sorry if the was not clarified well in the request.

**Comment by Riajyuu** [comment]
Do you mean that Dopmine accesses system theme settings?

**Comment by tjhrulz** [comment]
Yes and write to them instead of just reading.

**Comment by FurkanKambay** [comment]
I don't think you can change the system setting. You could go as far as launching the settings app with this URI: `ms-settings:colors`
Edit: Oh, wait, I thought this was a UWP app for a second. It could be changed by registry I suppose.

**Comment by Magentaize** [comment]
As an application, it shouldn't modify system settings.

**Comment by Riajyuu** [comment]
> As an application, it shouldn't modify system settings.

Though it is up to @digimezzo to decide but, IMHO, it can be also acceptable if Dopamine claims changes detail well & if provides option to restore them by ending/pausing playing.

**Comment by tjhrulz** [comment]
I will have to look up the API but I have seen other apps temporarily override the new theming system used in 8 and up.
